contrib/jenkins_tests/globals.sh

```diff
@@ -64,6 +64,18 @@ function do_github_status()
     "https://api.github.com/repos/$repo/statuses/${sha1}?access_token=$token"
 }
 
+function check_env()
+{
+	if [ $(git rev-parse --abbrev-ref HEAD | grep 'master' >/dev/null 2>&1 || echo $?) ]; then
+		if [ $(ofed_info -s | grep 'MLNX_OFED_LINUX-3.2' >/dev/null 2>&1 || echo $?) ]; then
+		    echo "environment [NOT OK]"
+		    exit 0
```

Comment:
you can use jenkins built-in env variables available during jenkins run:

http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/130/injectedEnvVars/